run_statement_of_liability_api_zap_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+environment="local"
+tags="not @smoke and not @ignore"
+if [ $# -gt 0 -a "$1" != "$environment" ];
+then
+  environment="$1"
+  tags="not @ignore"
+fi
+
+echo "*** running on $environment for tags '$tags' ***"
+
+sbt -Denvironment="$environment" -Dcucumber.options="--tags '$tags'" clean 'testOnly uk.gov.hmrc.test.cucumber.runners.StatementOfLiabilityApiTestRunner'
```

Comment:
Do the SOL acceptance tests run as part of the Zap tests?

Reply:
zap proxy listens to SOL acceptance running and then it does a scan after that wards